src/node.h

```diff
@@ -1002,7 +1002,7 @@ class InternalCallbackScope;
  */
 class NODE_EXTERN CallbackScope {
  public:
-  CallbackScope(v8::Isolate* isolate,
+  CallbackScope(Environment* env,
```

Comment:
I think the public addon-facing API should still use `v8::Isolate*`, because all addon-facing APIs use `v8::Isolate*`. You probably want to add a second constructor overload (as semver-minor instead of semver-major) instead of replacing this completely.

Reply:
@addaleax added the overload instead, PTAL